File: android/patient_reception/app/src/main/java/jp/ac/tottori_u/med/hosp/patient_reception/common/util/DateUtil.java
package jp.ac.tottori_u.med.hosp.patient_reception.common.util;

import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * Created by nishimura on 2018/10/17.
 */

public class DateUtil {

    public enum DateFormat {
        // GPS
        YYYYMMDD_HHMMSS("yyyy-MM-dd HH:mm:ss");

        private final String text;

        private DateFormat(final String text) {
            this.text = text;
        }

        public String getString() {
            return this.text;
        }
    }


    public static String date2String (Date date, DateFormat format) {
        String str = new SimpleDateFormat(format.getString()).format(date);
        return str;
    }

}

File: ios/patient_reception/patient_reception/view/Common/Util/DateUtil.swift
//
//  DateUtil.swift
//  patient_reception
//
//  Created by nishimura on 2018/10/09.
//  Copyright © 2018年 jp.ac.tottori-u.med.hosp. All rights reserved.
//

import UIKit

class DateUtil: NSObject {
    enum Format:String {
        case ymdHis = "yyyy-MM-dd HH:mm:ss"
    }
    
    public static func date2String (date:Date, format:DateUtil.Format) -> String{
        let formatter = DateFormatter()
//        formatter.dateFormat = DateFormatter.dateFormat(fromTemplate: format.rawValue, options: 0, locale: Locale(identifier: "ja_JP"))
        formatter.dateFormat = format.rawValue
        let dateString = formatter.string(from: date)
        
        return dateString
    }

}
